Angular-CLI/README.md

```diff
@@ -131,7 +131,7 @@ Then click on the gear icon to configure a launch.json file, selecting **Chrome*
 
 - Set a breakpoint in **app.component.ts** on the line that sets the `title` property of `AppComponent`.
 
-- Go to the Debug view, select the **'ng serve'** configuration, then press F5 or click the green play button.
+- Go to the Debug view, select the **'ng serve'** configuration, then press F5 or click the green play button to start 'Angular Live Development Server'.
```

Comment:
In PowerShell, when I execute `npx ng serve`, this output follows:
```shell
E:\demo\demo8 > npx ng serve
** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **

Date: 2019-03-05T04:02:09.634Z
Hash: e8203a6fa360bd3c865e
Time: 14749ms
chunk {0} 0.js, 0.js.map () 16.6 kB  [rendered]
chunk {1} 1.js, 1.js.map () 12.8 kB  [rendered]
...
```
I'm assuming the Angular team officially refers to this server as 'Angular Live Development Server' hence the capitalized letters. Although it may be slightly lengthy, my objective was to eliminate any confusion by being explicit. As you may experienced when being exposed to new frameworks, any vague documentation, at least for me, creates more uncertainties as I progress into development. 

Let me know if any more is needed, and again you may edit it as you wish.

Reply:
I see, thanks for clarifying. Reading the guide without knowing much about Angular I thought the guide referred to `Angular Live Development Server` as something within VS Code.